crates/merkle-tree/src/state_tree.rs

```diff
@@ -114,25 +114,22 @@ impl<'tx> StorageCommitmentTree<'tx, '_> {
 ///
 /// This tree maps a class's [SierraHash] to its [CasmHash]
 pub struct ClassCommitmentTree<'tx, 'queries> {
-    // FIXME(v0.11.0): This may be Poseidon hash depending.
     tree: MerkleTree<RcNodeStorage<'tx, 'queries>, PoseidonHash>,
 }
 
 impl<'tx> ClassCommitmentTree<'tx, '_> {
+    const LEAF_VERSION: stark_curve::FieldElement = stark_curve::FieldElement::ZERO;
+
     pub fn load(transaction: &'tx Transaction<'tx>, root: ClassCommitment) -> anyhow::Result<Self> {
         // TODO: migration to support this.
         let tree = MerkleTree::load("tree_class", transaction, root.0)?;
 
         Ok(Self { tree })
     }
 
-    pub fn get(&self, class: SierraHash) -> anyhow::Result<Option<CasmHash>> {
-        let value = self.tree.get(class.view_bits())?;
-        Ok(value.map(CasmHash))
-    }
-
```

Comment:
Why is it removed?

Reply:
Because now that we store a _hash_ of the (CASM hash, leaf version) tuple what's the point of getting that hash value for the Sierra hash?